utils/arp-whisper/Makefile

```
@@ -0,0 +1,52 @@
+# SPDX-License-Identifier: GPL-3.0-only
+#
+# Copyright (C) 2023 Facundo Acevedo
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=arp-whisper
+PKG_VERSION:=0.1.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/FacundoAcevedo/$(PKG_NAME)/archive/refs/tags/v$(PKG_VERSION).tar.gz?
```

```suggestion
PKG_SOURCE_URL:=https://codeload.github.com/FacundoAcevedo/arp-whisper/tar.gz/v$(PKG_VERSION)?
```

It's recommended to use codeload to be consistent with the other packages in the tree